src/evict/evict_lru.c

```diff
@@ -275,7 +277,7 @@ __wt_evict_server_wake(WT_SESSION_IMPL *session)
           bytes_updates);
     }
 
-    __wt_cond_signal(session, cache->evict_cond);
+    __wt_cond_signal(session, evict->evict_cond);
```

Comment:
nit:
```suggestion
    __wt_cond_signal(session, S2C(conn)->evict->evict_cond);
```
and drop the local `evict` variable

Reply:
Updated in [a544057](https://github.com/wiredtiger/wiredtiger/pull/11037/commits/a5440573ac1f3c62af793041ae953a033d885ae3)